src/openms/include/OpenMS/FILTERING/ID/IDFilter.h

```diff
@@ -336,6 +336,62 @@ namespace OpenMS
     /// Is the list of peptide evidences of this peptide hit empty?
     struct HasNoEvidence;
 
+    
+    /**
+       @brief Filter Peptide Hit by its digestion product
+
+    */
+
+    struct PeptideDigestionFilter
+    {
+
+      typedef PeptideHit argument_type;
+
+      EnzymaticDigestion& digestion_;
+      Int min_cleavages_;
+      Int max_cleavages_;
+      
+      PeptideDigestionFilter(
+        EnzymaticDigestion& digestion) :  
+      digestion_(digestion)
+      {}
+
+      inline void setMinCleavagesAllowed(const Int min_cleavages){ min_cleavages_ = min_cleavages; }
```

Comment:
The functions are used exactly once during the duration of the object. So we might as well set the value in the C'tor. Allow defaults (-1) if you want.
--> remove getter/setter